osext_procfs.go

```diff
@@ -27,8 +27,6 @@ func executable() (string, error) {
 		return execpath, nil
 	case "netbsd":
 		return os.Readlink("/proc/curproc/exe")
-	case "openbsd", "dragonfly":
```

Comment:
Oops, you're right, I missed it :-)
I pushed a new commit that bring dragonfly back.